data/scripts.js

```diff
@@ -108,7 +108,7 @@ let BattleScripts = {
 			}
 			// Dancer activates in order of lowest speed stat to highest
```

Comment:
```suggestion
			// Dancer activates in order of lowest speed stat to highest
			// Note that the speed stat used is after any volatile replacements like Speed Swap,
			// but before any multipliers like Agility or Choice Scarf
```

Reply:
@Marty-D is this accurate?

Reply:
Sounds right, yeah.

Reply:
It also occurs to me that the `.speed` stat test failure could be due to midgame speed changes?